src/cdk/overlay/overlay-container.ts

```diff
@@ -10,20 +10,27 @@ import {DOCUMENT} from '@angular/common';
 import {Inject, Injectable, OnDestroy} from '@angular/core';
 import {Platform} from '@angular/cdk/platform';
 
-declare const __karma__: unknown;
-declare const jasmine: unknown;
-declare const jest: unknown;
-declare const Mocha: unknown;
+// Avoid using `declare const` because it caused conflicts inside Google
+// with the real typings for these symbols.
+declare interface TestGlobals {
```

Comment:
I think you can drop the `declare` from the interface.

Reply:
You are correct in general, but this is a specific Closure compiler interop thing:

https://github.com/angular/tsickle/blob/master/README.md#differences-from-typescript

Reply:
We may want to leave a comment in that case since I can see it being removed in the future.

Reply:
Agreed, expanded the comment